code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts

```diff
@@ -2,6 +2,7 @@ import path from 'path';
 import { DefinePlugin, HotModuleReplacementPlugin, ProgressPlugin, ProvidePlugin } from 'webpack';
 import type { Configuration } from 'webpack';
 import HtmlWebpackPlugin from 'html-webpack-plugin';
+// @ts-ignore
```

Comment:
This started throwing an error at build, no idea why.

I tried installing the `@types/..` package but it's not using the right version of webpack, so it's un-usable..